custom_components/toyota/manifest.json

```
@@ -7,6 +7,6 @@
   "documentation": "https://github.com/DurgNomis-drol/ha_toyota",
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/DurgNomis-drol/ha_toyota/issues",
-  "requirements": ["mytoyota==2.1.1", "arrow"],
+  "requirements": ["mytoyota>=2.1.2,<3.0", "arrow"],
```

`2.1.4` fixes a recent issue in the API.
```suggestion
  "requirements": ["mytoyota>=2.1.4,<3.0", "arrow"],
```